PowerShell/Module/Private/_DeploymentFunctions.ps1

```diff
@@ -194,7 +194,15 @@ function _deployProject
 
         $amazonLambdaToolsPath = _configureAmazonLambdaTools
 
-        $env:AWS_EXECUTION_ENV="AWSLambdaPSCore"
+        $moduleVersion = "Unknown"
+        $module = $MyInvocation.MyCommand.Module
+        if ($module) {
+            $moduleVersion = $module.Version.ToString()
+        }
+        # The leading and trailing whitespaces are intentional
+        $userAgent = " lib/AWSLambdaPSCore#$moduleVersion "
```

Comment:
What is the intention of the trailing whitespaces? I want to avoid situations where there are more then one whitespace in a row as much as possible.

Reply:
Removed the trailing whitespace after syncing up internally.